src/App.tsx

```diff
@@ -10,20 +10,28 @@ import 'react-datepicker/dist/react-datepicker.css';
 import './components/style/elements/datepicker/datepicker-effekt.css';
 import 'react-toastify/dist/ReactToastify.css';
 import './components/style/elements/react-table/base.css';
-import { DEV_ENVIRONMENT } from "./config/config";
+import {
+  DEV_ENVIRONMENT,
+  AUTH_DOMAIN,
+  AUTH_CLIENT_ID,
+  AUTH_AUDIENCE,
+  AUTH_SCOPE,
+} from './config/config';
 import 'react-loading-skeleton/dist/skeleton.css';
 import { Auth0Provider } from '@auth0/auth0-react';
 
 const App: React.FC = () => {
   return (
     <Provider store={store}>
       <Auth0Provider
-        domain={'gieffektivt.eu.auth0.com'}
-        clientId={'jg0jOryc8mhJ0Z0FVLfrDxx5qaznd2x8'}
-        audience="https://data.gieffektivt.no"
-        scope="openid profile email read:donations write:donations read:profile write:profile read:distributions write:distributions read:agreements write:agreements read:vipps_api write:vipps_api admin"
+        domain={AUTH_DOMAIN}
+        clientId={AUTH_CLIENT_ID}
+        audience={AUTH_AUDIENCE}
+        scope="admin"
```

Comment:
Turns out this is the only one that's needed. `openid profile email` gets added automatically and the rest custom permissions are covered by the admin role

Example :point_down: 
<details>

```json
{
  "https://gieffektivt.no/user-id": 27,
  "https://gieffektivt.no/roles": [
    "admin",
    "donor"
  ],
  "iss": "https://gieffektivt.eu.auth0.com/",
  "sub": "auth0|63ff5798be60c526f4d3ac3a",
  "aud": [
    "https://data.gieffektivt.no",
    "https://gieffektivt.eu.auth0.com/userinfo"
  ],
  "iat": 1683296164,
  "exp": 1683382564,
  "azp": "jg0jOryc8mhJ0Z0FVLfrDxx5qaznd2x8",
  "scope": "openid profile email admin",
  "permissions": [
    "admin",
    "read:agreements",
    "read:distributions",
    "read:donations",
    "read:profile",
    "read:vipps_api",
    "write:agreements",
    "write:distributions",
    "write:donations",
    "write:profile",
    "write:vipps_api"
  ]
}
```